python/src/worker/tasks/factory.py

```diff
@@ -36,7 +35,7 @@ class TaskFactory:
             GetDoubletScore,
             GetMitochondrialContent,
             MarkerHeatmap,
-            GetExpressionCellSets
+            GetExpressionCellSets,
```

Comment:
I guess this was added by linter?

Reply:
we should add the same linter/formatter as in other projects like utils & inframock. You can see the setup in:
`biomage-utils/biomage-utils.code-workspace`
That file needs the following development dependencies which and config files:
```
dev-requirements.txt
setup.cfg
isort.cfg
```
and they are installed with `make develop` (there are also some other convenient commands like `make fmt` or `make check` but if you use visual studio it will be automatically formatted and checked).

Let me know if you need help setting it up, should be copy-paste.

Reply:
Yes